bigquery/google/cloud/bigquery/schema.py

```diff
@@ -14,6 +14,27 @@
 
 """Schemas for BigQuery tables / queries."""
 
+from google.cloud.bigquery_v2 import types
+
+
+# SQL types reference:
+# https://cloud.google.com/bigquery/data-types#legacy_sql_data_types
+# https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types
+LEGACY_TO_STANDARD_TYPES = {
+    "STRING": types.StandardSqlDataType.STRING,
+    "BYTES": types.StandardSqlDataType.BYTES,
+    "INTEGER": types.StandardSqlDataType.INT64,
```

Comment:
We should have aliases for the standard SQL names, as well, since the backend also supports those. e.g. a row from "INT64" to types.StandardSqlDataType.INT64

Reply:
For some reason I thought only the legacy names are used here, will update.